cpacker/MemGPT
# Adding pyautogen as optional dependency in pyproject.toml

**Please describe the purpose of this pull request.**
Is it to add a new feature? Is it to fix a bug?
Fixing the bug related while setup of the repository.

**How to test**
How can we test your PR during review? What commands should we run? What outcomes should we expect?
Setup a fresh new project in any IDE, autogen dependency was not met. After running "poetry add pyautogen --optional" command dependency was resolved.

**Have you tested this PR?**
Have you tested the latest commit on the PR? If so please provide outputs from your tests.
Yes. After running "poetry add pyautogen" dependency was resolved. Autogen dependency was not found in pyproject.toml file

**Related issues or PRs**
Please link any related GitHub [issues](https://github.com/cpacker/MemGPT/issues) or [PRs](https://github.com/cpacker/MemGPT/pulls).

**Is your PR over 500 lines of code?**
No

**Additional context**
Add any other context or screenshots about the PR here.
